tests/projects/failing-multi-repl-cabal-project/multi-repl-cabal-fail/multi-repl-cabal-fail.cabal

```
@@ -0,0 +1,23 @@
+cabal-version:       >=2.0
```

```suggestion
cabal-version:       3.0
```

Is `>=2.0` the correct syntax?

---

I think it is, 
`cabal-version:       >=2.0` occur in a lot of places in the tests. I admit I just copy it from https://github.com/haskell/hie-bios/blob/0e7f9f5796646c9107d6638d6e68367aa5a7cfe1/tests/projects/multi-cabal/multi-cabal.cabal. 😅